dashboard/app/models/levels/script_completion.rb

```diff
@@ -10,7 +10,7 @@
 #  level_num             :string(255)
 #  ideal_level_source_id :integer          unsigned
 #  user_id               :integer
-#  properties            :text(65535)
+#  properties            :text(16777215)
```

Comment:
Is this level type still used?

Reply:
there is no remaining level file, but the level still exists in levelbuilder, prod, and possibly other environments. seems worth cleaning up.